kolibri/plugins/learn/assets/src/vue/card/index.vue

```diff
@@ -0,0 +1,62 @@
+<template>
```

Comment:
I like this abstraction.

I keep forgetting about slots, but they seem to be the bestish way to allow for 'subclassing' of components, right?